README.md

```diff
@@ -77,27 +77,52 @@ Now that *oodeel* is installed, here are some basic examples of what you can do
 Load in-distribution and out-of-distribution datasets.
 
 ```python
-from oodeel.datasets import OODDataset
-
-ds_in = OODDataset(
-  'mnist', load_kwargs={"split":"test"},
-  backend="tensorflow").prepare(batch_size) # use backend="torch" if you prefer torch.DataLoader
-ds_out = OODDataset(
-  'fashion_mnist', load_kwargs={"split":"test"},
-  backend="tensorflow").prepare(batch_size)
+from oodeel.datasets import load_data_handler
+
+data_handler = load_data_handler("torch") # use backend="tensorflow" if using tensorflow
```

Comment:
Backend "tensorflow" requires different `load_kwargs` in `load_dataset` than "torch". As is, I understand that only changing the backend here is enough to run the code below.
I suggest to specify that `load_kwargs` must be also changed (or add a complete tensorflow code snippet)